__tests__/sources.ts

```diff
@@ -928,6 +929,38 @@ query Source($id: ID!) {
       'NOT_FOUND',
     ));
 
+  it('squad should have post moderation disabled', async () => {
```

Comment:
These test only tests typeorm really and what we want to test is mutations which actually change the data. It would be better that this test is inside createSquad mutation and tests that moderationRequired is disabled by default. It would also confirm backward compatibility.

Reply:
You can also add other case that Lee mentioned that when you pass `moderationRequired=true` its set correctly.